.delivery/build_cookbook/attributes/default.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Cookbook Name:: build_cookbook
+# Attributes:: default
+#
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#########################################################################
+# Terraform
+#########################################################################
+
+default['build-cookbook']['terraform_version'] = '0.8.1'
+
+#########################################################################
+# AWS
+#########################################################################
+
+default['build-cookbook']['aws_profile']     = 'chef-cd'
+default['build-cookbook']['aws_key_name']    = 'cd-infrastructure'
+default['build-cookbook']['aws_private_key'] = File.join(node['delivery']['workspace_path'], '.ssh', "#{node['build-cookbook']['aws_key_name']}.pem")
```

Comment:
These attributes won't be needed after https://github.com/chef-cookbooks/expeditor-build/pull/23 is merged.